tests/pipelines/test_pipelines_image_classification.py

```diff
@@ -177,6 +177,20 @@ def test_custom_tokenizer(self):
 
         self.assertIs(image_classifier.tokenizer, tokenizer)
 
+    @require_torch
+    def test_low_precision_pipeline(self):
```

Comment:
Let's update to a more specific test name and then add an equivalent torch.bfloat16 

```suggestion
    def test_torch_float16_pipeline(self):
```